samples/emoji-search/presenter/build.gradle

```diff
@@ -5,12 +5,19 @@ kotlin {
   js {
     browser()
   }
+  jvm {
+  }
 
   sourceSets {
     commonMain {
       dependencies {
         api projects.samples.emojiSearch.schema.compose
       }
     }
+    commonTest {
```

Comment:
Everything’s in one module for now. I think I should explode this into a few modules:

 * `redwood-testing`: RedwoodTester, MutableWidget, WidgetValue, ManualCoroutineDispatcher
 * `samples/emoji-search/widget/values`: the generated code currently in `EmojiSearchWidgets.kt`. This will depend on `redwood-testing`.

We could further split testing stuff from values stuff. I’d like a use case for values-without-testing first.

Reply:
I would put it at `redwood-compose-testin` since it's specifically a widget backend for the compose stuff (and the compose modules already depend on the widget modules).

Reply:
Can do.

Reply:
Done.

Reply:
I still need to move out the generated code...